test/tasks/MultisigTask.t.sol

```diff
@@ -243,14 +243,84 @@ contract MultisigTaskUnitTest is Test {
         assertEq(data, expectedData, "Wrong aggregate calldata");
     }
 
-    function testNonceStateOverrideApplied() public {
-        string memory taskConfigFilePath = "test/tasks/mock/configs/NonceStateOverrideConfig.toml";
-        runTestSimulation(taskConfigFilePath);
+    string constant commonToml = "l2chains = [{name = \"OP Mainnet\", chainId = 10}]\n" "\n"
+        "templateName = \"DisputeGameUpgradeTemplate\"\n" "\n"
+        "implementations = [{gameType = 0, implementation = \"0xf691F8A6d908B58C534B624cF16495b491E633BA\", l2ChainId = 10}]\n";
+
+    function testNonceAndThresholdStateOverrideApplied() public {
```

Comment:
Using inline testing to reduce bloat by creating more tests and possibility of interference by other tests using the same config.toml file.